investment-time/research.md

```diff
@@ -0,0 +1,9 @@
+# Research
+
+Investment time was proposed, piloted, and recommended outside of the normal
+scope of R&D. As such, artifacts were not drafted in GitHub. They are linked
+here for posterity.
+
+- 🤔 [Proposal](https://docs.google.com/document/d/1HXm9r4afnQGgz3CcddryoS2Al-YqU6I__1zuGtqhr3I/edit?usp=sharing)
```

Comment:
This seems to go to the same link as `Recommendation of adoption`, is that intentional?

Reply:
Whoops, wrong link! Fixing.